eip-5478.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 5478
+title: Reducing the gas cost of contract creation with existing code
+author: Qi Zhou (@qizhou)
+discussions-to: https://ethereum-magicians.org/t/eip-5478-reducing-the-gas-cost-of-contract-creation-with-existing-code/10419
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-08-17
+requires: 1014, 2929
```

Comment:
Does this require EIP-1014? I don't see any references.

Reply:
Yes, it depends on EIP-1014, which introduce 'CREATE2'.  Let me add a directly reference in the EIP.

Reply:
A direct reference is added at https://github.com/ethereum/EIPs/pull/5478/files#diff-750b1a09c7ed6095f759be1ff9b7f67e73a7a509aa58d660ff6dff631814a5c9R34